solidity/contracts/converter/LiquidTokenConverter.sol

```diff
@@ -212,15 +212,15 @@ contract LiquidTokenConverter is BancorConverter {
         (uint256 amount, uint256 fee) = purchaseRate(_depositAmount);

```

Comment:
@yudilevi : See previous comment (in `BancorFormula.sol`) about revert messages in read-only functions.
This is an example of one of the things that we will need to verify.
If you recall the GitHub issue that I've opened - it works fine in Ganache, but fails when tested on Etherscan (either because of their nodes, or because they're using web3.js).
We will need to executes this transaction with invalid `_depositAmount`, and make sure that it reverts correctly.